api/lib/application/scenarios-simulator/scenario-simulator-controller.js

```diff
@@ -30,12 +31,17 @@ async function simulateFlashAssessmentScenario(
     challengePickProbability,
     challengesBetweenSameCompetence,
     limitToOneQuestionPerTube,
+    minimumEstimatedSuccessRateRanges: minimumEstimatedSuccessRateRangesDto,
```

Comment:
Pinaillage : aligner les noms ?

```suggestion
    minimumEstimatedSuccessRateRanges,
```

Reply:
Je comprends que ça peut faire conflit avec `const minimumEstimatedSuccessRateRanges = _minimumEstimatedSuccessRateRangesToDomain(
    minimumEstimatedSuccessRateRangesDto,
  );` mais je pense qu'en fait le param de fonction ci-dessus devrait s'appeller `minimumEstimatedSuccessRateRangesConfig`
  
  car c'est la config ingérée par `_minimumEstimatedSuccessRateRangesToDomain` pour créer le `minimumEstimatedSuccessRateRanges` si j'ai bien compris